inst/scripts/importPDB.R

```diff
@@ -0,0 +1,158 @@
+# importPDB.R
+# Non-exported function
+
+#=============================STOP & SAVE======================================
+# Provide a mechanism to stop:

```

Comment:
Do you think this will work as a way of stopping the process and continuing it later? I think it will!